File: Easy/18.LetterCountI/solve.rb
def LetterCountI(str)

  top, top_w = 0, ""
  arr = str.split(' ')
  
  arr.each do |word|
     word.chars do |c|
        x = word.count(c)
        top = x and top_w = word if x > top
     end
  end
  
  return top_w if top > 1
  -1       
end
   
# keep this function call here    
puts LetterCountI(STDIN.gets)

File: Medium/14.Consecutive/solve.py
def Consecutive(arr): 
    arr = list(sorted(arr))
    numbers = []

    for i in range(arr[0] + 1, max(arr)):
		 
	    if i not in arr:
			numbers.append(i)

    return len(numbers)
 
    
# keep this function call here  
print Consecutive(raw_input())